.env.sample

```diff
@@ -38,4 +38,11 @@ ML_PROJECT_SERVICE_URL = "http://ml-project-service:3000"
 USER_SERVICE_URL = "http://user-service:3000"                                   // Base url of the sunbird enviornment
 CSV_REPORTS_PATH = "public/report"                                              // Report path
 
-APP_PORTAL_BASE_URL = "https://dev.sunbirded.org"
\ No newline at end of file
+APP_PORTAL_BASE_URL = "https://dev.sunbirded.org"
+
+#CACHE TIME TO LIVE                                                             //Cache data time to live
+CACHE_TTL = 43200
```

Comment:
@VISHNUDAS-tunerlabs mention that it is to be given in seconds

Reply:
@VISHNUDAS-tunerlabs CACHE_TTL defined in common constants.